src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -24,8 +29,20 @@
     const ENABLE_MAX_DEPTH = 'enable_max_depth';
     const DEPTH_KEY_PATTERN = 'depth_%s::%s';
 
+    /**
+     * @var PropertyTypeExtractorInterface|null
+     */
```

Comment:
The docblock is not needed (as the property is private, IDEs should be able to infer the type from the constructor).